docs/user/openstack/install_upi.md

```diff
@@ -572,7 +594,19 @@ The main file needs to be uploaded to an HTTP(S) location the Bootstrap node wil
 
 You are free to choose any storage you want. For example:
 
-* Swift Object Storage with tempurl
+* Swift Object Storage
+  Create the `<container_name>` container and upload the `bootstrap.ign` file:
+  ```sh
+  $ swift upload <container_name> bootstrap.ign
```

Comment:
Could you add the step for creating the container here this too?

Reply:
The container will also be created when running this command.

Reply:
Oh sweet! Sorry, I hadn't realised.